docs/guides/operator/advanced-configuration.adoc

```diff
@@ -204,6 +204,25 @@ It is achieved by providing certain JVM options.
 
 For more details, see <@links.server id="containers" />.
 
+=== Scheduling
+
+You may control several aspects of the server Pod scheduling via the Keycloak CR. The scheduling stanza exposes the standard Kubernetes affinity, tolerations, topology spread constraints, and the priority class name to fine tune the scheduling and placement of your server Pods. By default if you do not specify a custom affinity your Pods will have an affinity for the same zone to prevent stretch clusters and an anti-affinity for the same node to improve availability.
+
+For example if you wish to set the priority class name:
+
+[source,yaml]
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  scheduling:
+    priortyClassName: custom-high
+  ...
+----
```

Comment:
As discussed, it'd be really good to include some references to the k8s guides for the settings - as links. Hope it will not decrease the docs CI stability for the external links tests. But the UX will be improved, so I'm in favor to do it :) And after the private discussion, I agree that these examples might not be directly included in the KC guide.

@shawkins WDYT?

Reply:
Added a snippet showing all fields used and a link to the kubernetes scheduling docs.